middleware/packet-forward-middleware/router/ibc_middleware.go

```diff
@@ -170,8 +170,8 @@ func (im IBCMiddleware) OnRecvPacket(
 
 	var data transfertypes.FungibleTokenPacketData
 	if err := transfertypes.ModuleCdc.UnmarshalJSON(packet.GetData(), &data); err != nil {
-		logger.Error("packetForwardMiddleware OnRecvPacketfailed to unmarshal packet data as FungibleTokenPacketData", "error", err)
-		return newErrorAcknowledgement(fmt.Errorf("failed to unmarshal packet data as FungibleTokenPacketData: %w", err))
+		logger.Debug("packetForwardMiddleware OnRecvPacket payload is not a FungibleTokenPacketData")
```

Comment:
including the error in the message would be nice in case it is expected to succeed here

Reply:
Done